src/test/ui/wf/wf-unsafe-trait-obj-match.stderr

```diff
@@ -31,8 +31,12 @@ LL | trait Trait: Sized {}
 error[E0038]: the trait `Trait` cannot be made into an object
   --> $DIR/wf-unsafe-trait-obj-match.rs:25:25
    |
-LL |     let t: &dyn Trait = match opt() {
-   |                         ^^^^^^^^^^^ `Trait` cannot be made into an object
+LL |       let t: &dyn Trait = match opt() {
```

Comment:
I actually think that this is more accurate than what we were doing before... I could potentially revert this one.